### PR TITLE
Change in tests to support Gemini user-repo/scan

### PR DESCRIPTION
**Pull request by GeetikaBatra**
endpoint changes

Signed Off by: Geetika Batra <gbatra@redhat.com>

### CODE REVIEW

File: integration-tests/features/src/utils.py [hunk 1]
```diff
@@ -99,3 +100,13 @@ def oc_run_command(*args):
     command.extend(args)
 
     return subprocess.check_output(command)
+
+
+def read_data_gemini():
+    """Help function to read sample payload."""
+    dir_path = os.path.dirname(os.path.realpath(__file__))
```
Comment (tisnik) [inline integration-tests/features/src/utils.py]: why is this needed? You can just use `data/scan-test-data.json` directly.

**Comment by tisnik** [comment]
Oh I see, thanks a lot @GeetikaBatra

**Comment by GeetikaBatra** [comment]
@tisnik. Your Welcome

**Comment by yzainee** [comment]
@GeetikaBatra The content passed in the test case are for npm where as the git url passed is for maven. Either change the payload or change the git repo

**Comment by GeetikaBatra** [comment]
@yzainee, Do I create a test repo for that or do you have one?

**Comment by yzainee** [comment]
https://github.com/heroku/node-js-sample

**Comment by GeetikaBatra** [comment]
@yzainee, PTAL

File: integration-tests/features/gemini.feature [hunk 2]
```diff
@@ -36,8 +36,6 @@ Feature: Gemini Analytics API
   Scenario: Check the Gemini API /api/v1/user-repo/scan response
     Given Gemini service is running
         And Gemini service git url is https://github.com/jitpack/maven-simple
```
Comment (yzainee) [inline integration-tests/features/gemini.feature]: whats the significance of this repo here? will it get over ridden by the node repo provided later?
cc: @tisnik

Comment (tisnik) [reply integration-tests/features/gemini.feature]: @yzainee better to ask Agam or @samuzzal-choudhury as they are authors.

Comment (abs51295) [reply integration-tests/features/gemini.feature]: We need to change this feature file accordingly, if we changed the endpoint behavior.

**Comment by GeetikaBatra** [comment]
[test]

**Comment by GeetikaBatra** [comment]
Merging this one so that e2e tests pass for Gemini and changes are rolled onto staging.
